src/ansys/dpf/core/operators/mesh/skin.py

```diff
@@ -24,6 +24,11 @@ class skin(Operator):
         skin element is added to the skin
         mesh if all its nodes are in the
         scoping.
+    duplicate_shell : bool, optional
+        If input mesh contains shell elements, output
+        mesh shell elements (boolean = 1) are
+        duplicated, one per each orientation,
+        or (boolean = 0) remain unchanged.
```

Comment:
Hey @GuillemBarroso just noticed this sentence. I think it should be rephrased and improved.